src/runtime/app.config.ts

```diff
@@ -473,12 +473,10 @@ const input = {
 
 const formGroup = {
   wrapper: '',
-  label: {
```

Comment:
Changing this introduces a breaking change.

Reply:
Looking for another solution without a breaking change then.

Reply:
@benjamincanac rollback and added a new condition for 'label' prop names to solve the bug and avoid any breaking change.